lib/fedex/request/tracking_information.rb

```diff
@@ -103,6 +107,21 @@ def invalid_tracking_number?(error_code)
         error_codes = %w{6035 6037 6070 6125 6135 6140 6145 6150 6172 6173 6174 6185 30020 30030 500195 500200 500205}
```

Comment:
i noticed 6035 is included in the [bugsnag list](https://app.bugsnag.com/stitch-fix-1/fashionthing/errors/5eb3967e8a31460018ffbf5c?filters%5Bevent.since%5D%5B%5D%5Btype%5D=eq&filters%5Bevent.since%5D%5B%5D%5Bvalue%5D=2020-05-09T07%3A00%3A00.000Z&filters%5Bevent.before%5D%5B%5D%5Btype%5D=eq&filters%5Bevent.before%5D%5B%5D%5Bvalue%5D=2020-05-11T07%3A00%3A00.000Z&filters%5Bsearch%5D%5B%5D%5Btype%5D=eq&filters%5Bsearch%5D%5B%5D%5Bvalue%5D=fedex&event_id=5eb8dc69005b25addf850000) but isn't explicitly called out in the PR description, is there anything else that needs to be addressed for this error code?

Reply:
Refer to my [first comment](https://github.com/stitchfix/fedex/pull/14#issuecomment-626869128), but tl;dr 6035 was already taken care of in a separate PR (#12)